source/views/sis/course-search/list.js

```diff
@@ -27,7 +27,7 @@ export class CourseSearchResultsList extends React.PureComponent<Props> {
 	keyExtractor = (item: CourseType) => item.clbid.toString()
 
 	renderSectionHeader = ({section: {title}}: any) => (
-		<ListSectionHeader title={parseTerm(title)} />
+		<ListSectionHeader title={parseTerm(title).toUpperCase()} />
```

Comment:
I probably liked these more in mixed-case? Generally speaking, list headers === mixed case, and tableview section headers === uppercase.

Reply:
not outdated

Reply:
Ok cool beans I agree